server/mergin/sync/public_api.yaml

```diff
@@ -460,6 +460,41 @@ paths:
         '404':
           $ref: '#/components/responses/NotFoundResp'
       x-openapi-router-controller: mergin.sync.public_api_controller
+  '/project/version/{project_id}/{version}':
```

Comment:
IMO , version**s** will be better. Based on paginated endpoint /project/versions/paginated ...